static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -709,6 +709,10 @@ $quote-mark: 35px;
             }
         }
     }
+    // A fixed height is being added as a hotfix until final design decisions made for ad placement, important overrides current JS
```

Comment:
overrides the JS?

Reply:
There's a js file that applies sticky behaviour to the MPU and generates a height, this css value just gives a fixed height to all the ad slots as a hotfix (so supersedes that js value), plan is to have further discussion next week on placement etc. and then the JS file can be looked at

Reply:
why not disable the height being set in the JS? it doesn't feel very safe to have a set of things all trying to set a height on something and have them override each other?